SDWebImage/Core/SDCallbackQueue.h

```diff
@@ -17,18 +17,16 @@ typedef NS_ENUM(NSUInteger, SDCallbackPolicy) {
     SDCallbackPolicyDispatch = 1,
     /// Ignore any async/sync and just directly invoke `block` in current queue (without `dispatch_async`/`dispatch_sync`)
     SDCallbackPolicyInvoke = 2,
-    /// Ensure callback in main queue (no gurantee on main thread). Do `dispatch_async` if the current queue is not main queue; else do invoke `block`. Never use `dispatch_sync`, suitable for general UI-related code
-    SDCallbackPolicySafeAsyncMainQueue = 3,
     /// Ensure callback in main thread. Do `dispatch_async` if the `NSThread.isMainTrhead == true` ; else do invoke `block`. Never use `dispatch_sync`, suitable for special UI-related code
```

Comment:
Seems the comment is not correct. The code implementation shows “When current thread is not main thread, do dispatch_async”